app/src/main/res/xml/preferences.xml

```diff
@@ -42,7 +42,7 @@
             android:positiveButtonText="@null"
             app:numColumns="5"
             app:itemLayout="@layout/dash_grid_item_color"
-            />
+            /> -->
```

Comment:
I would just remove but it doesn't really matter.

Reply:
I removed this and the related classes. Turns out `ColorPreference` was also "borrowed" from AOSP.